src/content/en/fundamentals/performance/resource-prioritization.md

```diff
@@ -0,0 +1,351 @@
+different types of resources based on how critical they might be. So, for
+example, a `<script>` tag in your page’s `<head>` would be loaded in Chrome at a
+**High** priority (below CSS, at **Highest**), but that priority would change to
+**Low** if it has the async attribute (meaning it can be loaded and run
+asynchronously).
+
+Priorities become important when investigating loading performance in your site,
+so beyond the usual techniques of
+[measuring](/web/fundamentals/performance/critical-rendering-path/measure-crp)
+and
+[analyzing the critical rendering path](/web/fundamentals/performance/critical-rendering-path/analyzing-crp),
+it’s useful to know Chrome’s priority for each resource. You can find that in
+the Network panel in Chrome Developer Tools. Here’s what it looks like:
+
+
+<figure>
+  <div class="aspect-ratio"
+       style="width: 1810px; --aspect-ratio-w: 1810; --aspect-ratio-h: 564">
+    <img src="images/res-prio-priorities.png"
+    alt="An example of how priorities are displayed in Chrome Developer Tools">
+  </div>
+  <figcaption><b>Figure 1</b>: Priorities in Chrome Developer Tools. You may
+  need to enable the Priority column by right-clicking on the column headers.
+  </figcaption>
+</figure>
+
+
+These priorities will give you an idea of how much relative importance the
+browser attributes to each resource. And remember that subtle differences are
+enough for the browser to assign a different priority; for example, an image
+that is part of the initial render will be prioritized higher than an image that
+starts offscreen. If you’re curious about priorities,
+[this article by Addy Osmani](https://medium.com/reloading/preload-prefetch-and-priorities-in-chrome-776165961bbf){: .external}
+digs a lot deeper into the current state of priorities in Chrome.
+
+So what can you do if you find any resources that are marked with a different priority than the one you’d want?
+
+This article dives into three different declarative solutions, which are all
+relatively new `<link>` types. If your resources are crucial to the user
+experience but are being loaded at too low a priority, you can try fixing that
+in one of two ways: preload or preconnect. And if on the other hand you’d like
+the browser to fetch some resources only when it’s done dealing with everything
+else, try prefetch.
+
+Let’s look at all three!
+
+## Preload
+
+`<link rel="preload">` informs the browser that a resource will be needed as
+part of the current navigation, and that it should start getting fetched as soon
+as possible. Here’s how you use it:
+
+    <link rel="preload" as="script" href="super-important.js">
+    <link rel="preload" as="style" href="critical.css">
+
+Most of this is probably what you’d expect, except perhaps for the “as”
+attribute. This allows you to tell the browser the type of the resource you’re
+loading, so that it can be handled correctly; in fact, the browser will not use
```

Comment:
Make these separate sentences, drop the 'in fact', and avoid future tense:  "...handled correctly. The browser doesn't use the preloaded resource unless...'

Reply:
Done.